frontend/Dockerfile

```diff
@@ -0,0 +1,22 @@
+# Create image based on the official Node image from dockerhub
+FROM node:lts-buster
+
+# Create app directory
+WORKDIR /usr/src/app
+
+# Copy dependency definitions
+COPY package.json /usr/src/app
+
+# Install dependecies
+RUN npm set progress=false \
+   && npm config set depth 0 \
+   && npm i install
```

Comment:
@LaxminarayanaV7416 Seems there is a typo in this command

Reply:
npm i install is a command, which is bascially similar to npm install or npm i, the twice we added to make sure its runs twice to make sure it didnt miss any packages, You know sometimes due to parallel installation it misses the packages this happens rarely!

Reply:
Thanks @LaxminarayanaV7416 It's working perfectly fine. Can you create Readme file and add Docker Commands into it before merging?